ganga/GangaGUI/gui/__init__.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask
+from GangaGUI.config import Config
+import sys
+
+# Disable development server warning (as it is not applicable to our GUI Flask App)
+cli = sys.modules['flask.cli']
+cli.show_server_banner = lambda *x: None
+
+# Flask App and get configuration from config.py file
+app = Flask(__name__)
+app.config.from_object(Config)
+
+from GangaGUI.gui import routes
```

Comment:
Put all imports at the start (unless there is an issue with it).

Reply:
In Flask, due to the circular import issue, we have to import routes are the bottom. (It has become a kind of a convention)